recipe/meta.yaml

```diff
@@ -9,44 +9,33 @@ source:
   sha256: 821aa2e5b709a2e5d3922dcf2c5d445b4898a6ef8bac683cfb00125eafbca029
 
 build:
-  number: 0
-  skip: true  # [not linux]
-  # cutlass 3.1 requires CUDA >=11.4
-  skip: true  # [not (cuda_compiler_version or "").startswith("12")]
+  number: 2
+  skip: true  # [osx or cuda_compiler_version not in ("12.0",)]
   run_exports:
     - {{ pin_subpackage('cutlass', max_pin='x.x') }}
```

Comment:
As this is header only, should we drop this? Or is there is still a need to align CUTLASS versions between packages somehow?

```suggestion
```

Reply:
I think it's a good point. Happy to simplify it!